File: app/src/main/java/org/hackillinois/android/model/Event.java
package org.hackillinois.android.model;

public class Event {
    private String name;
    private String description;
    private long startTime;
    private long endTime;

    public Event(String name, String description, long startTime, long endTime) {
        this.name = name;
        this.description = description;
        this.startTime = startTime;
        this.endTime = endTime;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/org/hackillinois/android/model/Event.kt
package org.hackillinois.android.model

data class Event (val name: String,
                  val description: String,
                  val startTime: Long,
                  val endTime: Long,
                  val locations: List<EventLocation>,
                  val sponsor: String,
                  val eventType: String
)

data class EventLocation (var description: String,
                          val latitude: Double,
                          val longitude: Double
)

val SiebelCenter = EventLocation("Siebel Center", 40.1138, -88.2249)
val EceBuilding = EventLocation("ECE Building", 40.1148, -88.2280)